projects/js-packages/components/components/button/stories/index.tsx

```diff
@@ -70,6 +70,12 @@ export default {
 				options: [ 'none', ...Object.keys( icons ) ],
 			},
 		},
+		variant: {
+			control: {
+				type: 'select',
+				options: [ 'primary', 'secondary', 'link', 'external-link' ],
```

Comment:
Should `external-link` still be an option here?

Reply:
Yes, we should keep it until we update all places where we are using`external-link` variation.

Reply:
there are a few places, though.

Reply:
agree, but let's emove it once we stop using the variation